crates/esplora/Cargo.toml

```diff
@@ -21,6 +21,9 @@ futures = { version = "0.3.26", optional = true }
 bitcoin = { version = "0.30.0", optional = true, default-features = false }
 miniscript = { version = "10.0.0", optional = true, default-features = false }
 
+[dev-dependencies]
+bdk_testenv = { path = "../testenv", default_features = false }
+
 [target.'cfg(not(target_arch = "wasm32"))'.dev-dependencies]
 electrsd = { version= "0.25.0", features = ["bitcoind_25_0", "esplora_a33e97e1", "legacy"] }
```

Comment:
Do we need to make the tests run in WASM? Because `electrsd` is a mandatory dependency of `bdk_testenv`.

Reply:
Right now for CI only only do a `cargo check` for WASM and don't rerun the tests for that target. Since I haven't heard of any WASM issues that we slipped through by not running the tests I don't think we need to change `TestEnv` to work with WASM.